i3_agenda/event.py

```diff
@@ -62,7 +62,8 @@ def is_urgent(self):
         return self.get_datetime() < urgent and not now > five_minutes_started
 
     def is_allday(self) -> bool:
-        return self.get_datetime().hour == 0
+        end_minus_24hrs = dt.datetime.fromtimestamp(self.end_time) - dt.timedelta(hours=24)
```

Comment:
what about the following case:  
I have a multiple all-day event (for example a trip) that lasts from Tuesday to Friday, it's still an all day event, but the diff wont be 24 hours, it would be 72 hours.  
If we're going with the time calculation solution instead of the old method, we could use the following algorithm:  

Consider an event an all day event if all the followings are true:  
- `if start_time.hour == 0 and start_time.minute == 0`  
- consider delta = end_time - start_time; `if delta % 24 == 0`   

In theory this solution should suffice.  
I'm actually not sure how Google represents start/end time of all day events and didn't have the chance to check it. If it works like I would expect (exactly 24 hours), the algorithm I proposed should work.   
What do you think?

Reply:
Yeaahh it's true, realy a better logic.

Yes, I looked at the events, and all day Google calendar events begins in the start day at midnight and ends at end day + 1 at midnight. So I yes, guess it looks to work.

I only changed the first condition to: `self.get_datetime().time() == dt.time(0)`